Kickstarter-iOS/Views/Cells/ThanksCategoryCell.swift

```diff
@@ -3,14 +3,48 @@ import Library
 import Prelude
 import UIKit
 
+internal protocol ThanksCategoryCellDelegate: AnyObject {
+  func thanksSeeAllProjectsTapped(_ cell: ThanksCategoryCell, category: KsApi.Category)
+}
+
 internal final class ThanksCategoryCell: UITableViewCell, ValueCell {
-  @IBOutlet fileprivate var seeAllProjectsButton: UIButton!
+  internal weak var delegate: ThanksCategoryCellDelegate?
+  fileprivate let viewModel: ThanksCategoryCellViewModelType = ThanksCategoryCellViewModel()
+
+  @IBOutlet fileprivate var seeAllProjectCategoryButton: UIButton!
+
+  override func awakeFromNib() {
+    super.awakeFromNib()
+
+    self.seeAllProjectCategoryButton
+      .addTarget(self, action: #selector(self.seeAllProjectCategoryTapped), for: .touchUpInside)
+  }
 
   func configureWith(value category: KsApi.Category) {
-    _ = self.seeAllProjectsButton
+    self.viewModel.inputs.configureWith(category: category)
+  }
+
+  internal override func bindStyles() {
+    super.bindStyles()
+
+    _ = self.seeAllProjectCategoryButton
       |> greyButtonStyle
-      |> UIButton.lens.title(for: .normal) %~ {
-        _ in Strings.See_all_category_name_projects(category_name: category.name)
+  }
+
+  internal override func bindViewModel() {
+    super.bindViewModel()
+
+    self.viewModel.outputs.notifyToGoToDiscovery
+      .observeForControllerAction()
+      .observeValues { [weak self] in
+        guard let _self = self else { return }
+        self?.delegate?.thanksSeeAllProjectsTapped(_self, category: $0)
```

Comment:
Here you could just call `_self.delegate?.thanksSeeAllProjectsTapped(_self: category: $0)` since you unwrap `self` in the previous line.